test/integration/stress.js

```diff
@@ -7,7 +7,7 @@ test('sb', function (t) {
     var set = data.sb;
     t.plan(set.length * 2);
     for (var i in data.sb) {
-        parser(data.sb[i], function (err, res) {
+        parser(data.sb[i], false, function (err, res) {
```

Comment:
Given the above comment, if you go for option 2 it would be good to keep some of these as-is to test the argument parsing / defaults.